src/utils/victim-identification/helpers/addresses.extractor.ts

```diff
@@ -0,0 +1,94 @@
+import { providers, ethers } from "ethers";
+import { EVM } from "evm";
+import { TransactionEvent } from "forta-agent";
+
+const CONTRACT_SLOT_ANALYSIS_DEPTH = 20;
```

Comment:
Theoretically, it can be in any slot, but as we have to choose an upper limit, I decided to follow the same as on the _Suspicious Contract Creation_ bot